### DISCUSSION TITLE
Why does the dashboard scrolling: true option sets all cards to the same size?

### DISCUSSION THREAD

**Discussion by alxsimon** [quarto-dev/quarto-cli#9858] [Q&A] [answered]
### Description

Hi,
When using the dashboard option `scrolling: true`, it seems that all cards on the page end up being set to the same height.
This is not ideal when having cards of very different sizes, for example if one card is way longer that others it makes a lot of white space in other cards.
Am I missing an option or something to let cards autosize the height?
Thanks

**Comment by alxsimon** [comment]
It seems having something like this works
````
```{python}
#| fill: false
```
````
but I couldn't find a global option to set this up

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us  to recreate your problem so that we can fix it: please help us help you! Thanks.




---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by alxsimon** [comment]
Hi here is a reproducible example where the second plot size is not auto-sized, the second plot needs `|# fill: false` to be dispalyed properly which I feel should be the default with `scrolling: true`?
````
---
title: "test"
format:
  dashboard:
    scrolling: true
---

# first page

```{python}
#| title: multiple plots
import matplotlib.pyplot as plt

for i in range(4):
	x = [1, 2, 3, 4, 5]
	y = [1, 4, 9, 16, 25]
	plt.plot(x, y)
	plt.show()
```

```{python}
#| title: one plots
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]
plt.plot(x, y)
plt.show()
```
````

quarto check
```txt
Quarto 1.5.29
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.29
      Path: /home/alexis/opt/quarto-1.5.29/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /usr/bin
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.3
      Path: /usr/bin/python3
      Jupyter: 5.7.2
      Kernels: python3, local_pcangsd, ir

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.4.0
      Path: /usr/lib64/R
      LibPaths:
        - /home/alexis/R/x86_64-redhat-linux-gnu-library/4.4
        - /usr/lib64/R/library
        - /usr/share/R/library
      knitr: 1.46
      rmarkdown: 2.26

[✓] Checking Knitr engine render......OK
```

**Reply by mcanouil** [reply]
The issue is that the cards are in a grid with `grid-template-rows` set. The cards inside actually have no height but they do fill the available height of the row.

**Reply by mcanouil** [reply, marked as answer]
I've opened an enhancement issue:

- #9907

**Reply by alxsimon** [reply]
thanks for the transfer and the details!
